clients/lsp-rust.el

```diff
@@ -926,6 +927,37 @@ or JSON objects in `rust-project.json` format."
 (lsp-defun lsp-rust--analyzer-debug-lens ((&Command :arguments? [args]))
   (lsp-rust-analyzer-debug args))
 
+(defun lsp-rust-analyzer--semantic-modifiers ()
```

Comment:
first, thanks a lot for your patience! Sorry the PR has been blocked so long. I'm fine with your PR as-is, though I wonder if there's a reason this is a `defun` as opposed to a variable, defcustom or something? Not trying to be annoying here, honest question :)

Reply:
The simple answer is I repurposed the prior version, and changed the function name to plug it into a new place.  I will take a look when I have time to make it something more idiomatic.